rfc3164.go

```diff
@@ -43,6 +43,56 @@ const (
 // Facility represents a syslog facility code
 type Facility int
 
+//FacilityTextToFacility accepts a string representation of a syslog
+// facility and returns a captainslog.Facility
+func FacilityTextToFacility(facilityText string) (Facility, error) {
+	var err error
```

Comment:
@bryanl will move to a map wrapped with a conditional key check in a follow up PR :thumbsup: